.storybook/modes.ts

```diff
@@ -0,0 +1,52 @@
+import { baseLocales } from "./i18next"
+import { chakraBreakpointArray } from "./preview"
+
+export const viewportModes = chakraBreakpointArray.reduce<{
+  [mode: string]: { viewport: string }
+}>((arr, [token]) => {
+  return {
+    ...arr,
+    [token]: {
+      viewport: token,
+    },
+  }
+}, {})
+
+export const langModes = Object.keys(baseLocales).reduce<{
+  [locale: string]: { locale: string }
+}>((arr, curr) => {
+  return {
+    ...arr,
+    [curr]: {
+      locale: curr,
+    },
+  }
+}, {})
+
+type LangViewModeObj = {
+  [key: string]: { viewport: string; locale: string }
+}
+
+export const langViewportModes = Object.entries(
+  viewportModes
+).reduce<LangViewModeObj>((arr, curr) => {
+  const [viewKey, viewVal] = curr
+
+  const currLangViewObj = {} as LangViewModeObj
+
+  Object.entries(langModes).forEach(([langKey, langVal]) => {
+    currLangViewObj[`${langKey}-${viewKey}`] = {
+      viewport: viewVal.viewport,
+      locale: langVal.locale,
+    }
+  })
+
+  return {
+    ...arr,
+    ...currLangViewObj,
+  }
+}, {})
+
+export const allModes = {
+  ...langViewportModes,
+}
```

Comment:
@TylerAPfledderer Do we need this? Looks like we only import `langViewportModes` elsewhere... Otherwise this lgtm!

Reply:
Good point. Examples in the docs show compiling different mode variations into one object like this to extract. But definitely not necessary here. I can remove it.

Reply:
@wackerow this is now removed.